Equipo2Hosteleria/src/main/java/Modelo/CarroCompra.java

```diff
@@ -0,0 +1,61 @@
+package Modelo;
+
+import java.util.Arrays;
+
+public class CarroCompra {
+	
+	public double TotalCarro=0;
+	public Object[][] array = new Object[0][4];
```

Comment:
Para esto, no es mejor que os creéis una clase, y utilizar un array de objetos de esa clase directamente?

Reply:
Ya está hecho.